Sprint-2/implement/tally.test.js

```diff
@@ -20,15 +20,31 @@ const tally = require("./tally.js");
 // When passed an array of items
 // Then it should return an object containing the count for each unique item
 
+test("passed an array of items it should return an object containing the count for each unique item", () => {
+  expect(tally(["a", "a", "b", "c"])).toEqual({ a: 2, b: 1, c: 1 });
+});
+
 // Given an empty array
 // When passed to tally
 // Then it should return an empty object
-test.todo("tally on an empty array returns an empty object");
+
+test("passed an array of items it should return an object containing the count for each unique item", () => {
+  expect(tally([])).toEqual({});
```

Comment:
If this test is testing the same thing as the previous one (they have the same description), 
you can combine them into one test, as in:
```
test("passed an array of items it should return an object containing the count for each unique item", () => {
  expect(tally(["a", "a", "b", "c"])).toEqual({ a: 2, b: 1, c: 1 });
  expect(tally([])).toEqual({});
});

```

Reply:
Thank you, I will use it)